test/ma_test_analytic_model.cc

```diff
@@ -171,10 +171,11 @@ apf::Mesh2* createSphereMesh()
 int main(int argc, char** argv)
 {
   MPI_Init(&argc,&argv);
-  PCU_Comm_Init();
+  {
+  auto PCUObj = std::unique_ptr<pcu::PCU>(new pcu::PCU(MPI_COMM_WORLD));
```

Comment:
Can a PCU constructor/factory method be created to do this?

Reply:
I think if we want to keep this as a `unique_ptr` `make_unique` should be used. Is there a reason to have a `unique_ptr` rather than just constructing a PCU object and passing the address to any of the core functions?

Reply:
If we could drop the unique_ptr that would be ideal; users will copy what is done in these examples and we should try to follow/present 'best practices'.